spec/ruby/core/fiber/raise_spec.rb

```diff
@@ -91,6 +91,23 @@
 
     fiber_two.resume.should == [:yield_one, :rescued]
   end
+
+  ruby_version_is "3.4" do
+    it "raises on the resumed fiber" do
+      root_fiber = Fiber.current
+      f1 = Fiber.new { root_fiber.transfer }
+      f2 = Fiber.new { f1.resume }
+      f2.transfer
+
+      error = nil
+      begin
+        f2.raise(RuntimeError)
+      rescue RuntimeError => error
+      end
+
+      error.should be_kind_of(RuntimeError)
```

Comment:
`-> { ... }.should raise_error(RuntimeError, "message")`.
Would also be good to give a message to `Fiber#raise`  to not accidentally catch another RuntimeError.

Reply:
Thanks, that's a good improvement.